spec/models/grant_submission_spec.rb

```diff
@@ -1,8 +1,8 @@
 describe GrantSubmission do
   subject { FactoryGirl.create(:grant_submission) }
 
-  describe '#max_funding_dollars' do
-    its(:max_funding_dollars) { is_expected.to eq(subject.grant.max_funding_dollars) }
+  describe '#funding_levels_csv' do
+    its(:funding_levels_csv) { is_expected.to eq('subject.grant.funding_levels_csv') }
```

Comment:
I think this original test was broken.  It should have been like "requested_funding_dollars is less than or equal to max_funding_dollars".  In this case, the test is much more complicated:  all of the integers in the funding_requests_csv must pass the funding_levels_csv.  The current logic lives in app/assets/javascripts/grant_submissions/new.js in validate_funding_levels.

all that said, I think right now we don't enforce the funding requests in ruby code at all other than checking to see that they've entered a list of integers (grant_submission.rb:115).  Maybe this test can just confirm that the funding_requests_csv passes validation?  I'm not sure why this is here at all (I didn't write most of the tests)